ui/tests/integration/components/clients/page-header-test.js

```diff
@@ -23,52 +25,56 @@ module('Integration | Component | clients/export-button', function (hooks) {
     this.startTimestamp = '2022-06-01T23:00:11.050Z';
     this.endTimestamp = '2022-12-01T23:00:11.050Z';
     this.selectedNamespace = undefined;
+    this.upgradesDuringActivity = undefined;
```

Comment:
should we set this to an empty array since that looks to be what`filterVersionHistory` returns if there's no relevant upgrades?